src/pages/index.js

```diff
@@ -26,13 +26,19 @@ const useStyles = makeStyles(theme => ({
     fontSize: 60,
     lineHeight: '71px',
     letterSpacing: '0.08em',
+    [theme.breakpoints.up('md')]: {
+      fontSize: 80,
+    },
   },
   diplomacyTextProperty: {
     color: '#fff',
     fontFamily: "'Rubik' , sans-serif",
     fontWeight: 'bold',
     fontSize: '24px',
     lineHeight: '28px',
+    [theme.breakpoints.up('md')]: {
+      fontSize: 35,
+    },
```

Comment:
also increase the `lineHeight` so that diplomacy and jecrc MUN will get a space between them

Reply:
Done